ee/clickhouse/sql/cohort.py

```diff
@@ -43,9 +43,9 @@
         SELECT id, argMax(properties, person._timestamp) as properties, sum(is_deleted) as is_deleted FROM person WHERE team_id = %(team_id)s GROUP BY id
     ) as person
     LEFT JOIN (
-        SELECT person_id FROM cohortpeople WHERE cohort_id = %(cohort_id)s AND team_id = %(team_id)s
+        SELECT person_id, sum(sign) AS sign FROM cohortpeople WHERE cohort_id = %(cohort_id)s AND team_id = %(team_id)s GROUP BY person_id
     ) as cohortpeople ON (person.id = cohortpeople.person_id)
-    WHERE cohortpeople.person_id = '00000000-0000-0000-0000-000000000000'
+    WHERE (cohortpeople.person_id = '00000000-0000-0000-0000-000000000000' OR sign = 0)
```

Comment:
Can `sum(sign)` be larger than 0?

Reply:
yes, but the condition i'm trying to cover here is if the person_id exists in the cohortpeople table but has been tomstoned (so a row with -1 and 1 both exist) we still consider it when looking for cohortpeople to insert. If it's already 1, we don't need to do anything to it because it will either stay part of the cohort or be cleaned up by the following `REMOVE_PEOPLE_NOT_MATCHING_COHORT_ID_SQL` query